mods/d2k/rules/infantry.yaml

```diff
@@ -52,6 +52,14 @@ engineer:
 	Captures:
 		CaptureTypes: building
 		PlayerExperience: 10
+		ValidRelationships: Enemy
```

Comment:
Why are we disabling the capture of neutral structures?

Reply:
sorry my mistake. Fixed